apps/web/src/pages/_document.tsx

```diff
@@ -11,6 +14,25 @@ export default function Document() {
         <link href="/assets/images/govuk-apple-touch-icon-152x152.png" rel="apple-touch-icon" sizes="152x152" />
         <link href="/assets/images/govuk-apple-touch-icon-167x167.png" rel="apple-touch-icon" sizes="167x167" />
         <link href="/assets/images/govuk-apple-touch-icon-180x180.png" rel="apple-touch-icon" sizes="180x180" />
+        <Script id="gtm" strategy="afterInteractive">
```

Comment:
What are the privacy implications of loading this script in all documents?
Can we load this script only when we have consent?

Do we require any functionality that is provided without consent?

Reply:
no privacy concerns, one of the first things this does is set the default state of the gtag permissions to deny all. no cookies are actually created until a user clicks accept. it does not delete the cookies if the user  already has them, but does remove the permissions for them to work.

however there is no functionality provided that is used without consent.